test/test_type_promotion.py

```diff
@@ -808,6 +808,48 @@ def test_integer_addcdiv_deprecated(self, device, dtype):
         with self.assertRaisesRegex(RuntimeError, '^Integer division.+is no longer supported+'):
             t.addcdiv_(t, t)
 
+    def test_addcdiv_promotion(self, device):
+        types = (
+            (torch.float64, torch.float64, torch.complex128),
+            (torch.long, torch.bfloat16, torch.float32),
+        )
+
+        for type1, type2, type3 in types:
+            # Cannot use rand with long.
```

Comment:
use `make_tensor` to construct test arguments

Reply:
done